packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -531,7 +531,12 @@ export default class Meeting extends StatelessWebexPlugin {
      * @private
      * @memberof Meeting
      */
-    this.locusInfo = new LocusInfo(this.updateMeetingObject.bind(this), attrs.locus ? attrs.locus : {}, this.webex, this.id);
+    this.locusInfo = new LocusInfo(this.updateMeetingObject.bind(this), this.webex, this.id);
+
+    // We had to add listeners first before setting up the locus instance
+    this.setUpLocusInfoListeners();
```

Comment:
Do we tear these down anywhere?

Reply:
Nope we dont currently we listen on meeting create and it gets destroyed on meeting removed